media/magicavoxel_vox.ksy

```diff
@@ -5,8 +5,9 @@ meta:
   endian: le
   license: MIT
   title: MagicaVoxel File
-doc-ref: 'https://ephtracy.github.io/ MagicaVoxel Homepage'
-doc-ref: 'https://github.com/ephtracy/voxel-model/blob/master/MagicaVoxel-file-format-vox.txt Format Description'
+doc-ref:
+  - https://ephtracy.github.io/ MagicaVoxel Homepage
+  - https://github.com/ephtracy/voxel-model/blob/master/MagicaVoxel-file-format-vox.txt Format Description
```

Comment:
This is only allowed in 0.9+. Please add relevant `ks-version: 0.9` too?

Reply:
Done!